pages/k8s/1.21/release-notes.md

```diff
@@ -0,0 +1,77 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Release notes"
+  description: Release notes for CDK
+keywords: kubernetes,  release, notes
+tags: [news]
+sidebar: k8smain-sidebar
+permalink: 1.21/release-notes.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+# 1.21
+
+### April XX, 2021 - [charmed-kubernetes-XXX](https://api.jujucharms.com/charmstore/v5/charmed-kubernetes-XXX/archive/bundle.yaml)
+
+## What's new
+
+- Azure Arc conformance
+
+Charmed Kubernetes is compliant with the Azure Arc Validation test suite.
+More information about this program can be found in the
+[azure-arc-validation documentation][arc-docs].
+
+- Container images by release
+
+[LP 1891530](https://bugs.launchpad.net/cdk-addons/+bug/1891530) describes an
+upgrade failure for deployments that use a private image registry. The Charmed
+Kubernetes release process now publishes a
+[list of required images per-release][images-per-release] for administrators
+to easily determine what registry changes are needed prior to an upgrade.
+
+## Component upgrades
+
+- cloud-provider-openstack 1.20.0
+- coredns 1.8.3
+- kube-state-metrics 1.9.8
+- kubernetes-dashboard 2.2.0
```

Comment:
i added both coredns and kube-dashboard here because cdk-addons *does* include these new versions... But we also have operator charms for them.  I'm not sure if it would confuse people that are using operators to see these listed here.

Reply:
I don't think so. I guess we will know if someone complains :wink: